finetune/encoding/input_encoder.py

```diff
@@ -66,7 +66,7 @@ def _remove_repeated_whitespace(encoded):
         encoded.token_ids, encoded.tokens, encoded.token_ends, encoded.token_starts
     ):
         mask = [
-            i != 0 and token == tokens[i - 1] == " " for i, token in enumerate(tokens)
+            i != 0 and token.strip() == tokens[i - 1].strip() == "" for i, token in enumerate(tokens)
```

Comment:
I don't know if we want to collapse newlines. Could we change this to `strip(" ")`?

Reply:
Updated it, but worth nothing that the fast tokenizers turn all whitespace into empty strings, so I don't  think we can avoid collapsing newlines with them. I guess we could slice the text and check that instead?